packer/scripts/ubuntu/install_docker.sh

```diff
@@ -5,21 +5,14 @@ set -o pipefail
 sudo apt-get -y update
 sudo apt-get install -y linux-image-extra-$(uname -r)
 
```

Comment:
The point of doing this manually rather than just "curl -sSL https://get.docker.com/ | sh" is to have the ability to lock to a given version and we are removing that here.
Can not we do "sudo apt-get install -y mesos=${MESOS_VERSION}" the something like "it { should be_installed.by('apt').with_version()"